lib/pages/RidePage.dart

```diff
@@ -441,39 +441,47 @@ class TimeLineRow extends StatelessWidget {
     ]);
 
     Widget locationCircle() {
-      return Container(
-          width: 26,
-          child: useCarIcon
-              ? SvgPicture.asset('assets/images/carIcon.svg')
-              : stopCircle);
+      return ExcludeSemantics(
+        child: Container(
+            width: 26,
+            child: useCarIcon
+                ? SvgPicture.asset('assets/images/carIcon.svg')
+                : stopCircle),
+      );
     }
 
-    return Row(children: [
-      locationCircle(),
-      SizedBox(width: 16),
-      text == null
-          ? infoWidget
-          : text == noShowMessage
-          ? Text(noShowMessage,
-          style: TextStyle(fontSize: 16, color: Color(0xFFF44336)))
-          : isCurrentRide
-          ? Text(text,
-          style: TextStyle(
-              fontSize: 16,
-              color: Colors.black,
-              fontWeight: FontWeight.bold))
-          : Text(text,
-          style:
-          TextStyle(fontSize: 16, color: CarriageTheme.gray4))
-    ]);
+    return Row(
+        children: [
+          locationCircle(),
+          SizedBox(width: 16),
+          text == null
+              ? infoWidget
+              : text == noShowMessage
+              ? Expanded(
+            child: Text(noShowMessage,
+                style: TextStyle(fontSize: 16, color: Color(0xFFF44336))),
+          )
+              : isCurrentRide
+              ? Expanded(
+            child: Text(text,
+                style: TextStyle(
+                    fontSize: 16,
+                    color: Colors.black,
+                    fontWeight: FontWeight.bold)),
+          ) : Expanded(
+            child: Text(text,
+                style:
+                TextStyle(fontSize: 16, color: CarriageTheme.gray4)),
+          )
+        ]);
   }
 }
 
```

Comment:
On the ride page the cancel ride button isn't announced as a button when using voiceover. Is that intended?

Reply:
Good catch, thanks! Will wrap with a Semantics so that it reads properly